query/src/sessions/session_ctx.rs

```diff
@@ -77,10 +79,21 @@ impl SessionContext {
         *lock = db
     }
 
-    pub fn get_tenant(&self) -> String {
+    pub fn get_current_tenant(&self) -> String {
+        if self.conf.query.management_mode {
```

Comment:
add a comment about the difference between the current tenant logic of management_mode and normal case?